cs130_front/src/components/views/ViewGroupPage/ViewGroupPage.js

```diff
@@ -1,31 +1,88 @@
 import React, { useState, useEffect } from 'react';
-import UserList from '../../UserList/UserList';
+import ClassList from '../../CurrentClasses/ClassList';
+import GroupItem from '../GroupsPage/GroupItem';
+import Text from '../../Text/Text';
 import '../../../App.css';
+import Button from '../../Button/Button';
+import * as Colors from '../../../constants/Colors';
+import CreateGroup from '../GroupsPage/CreateGroup';
 
 export default function ViewGroupPage(props) {
-  const classes = [
-    {name: "Discrete Mathematics", courseId: 1, groups: [{name: "DM Squad", groupId: 1}, {name: "DM Squad II", groupId: 1}] }, 
-    {name: "Computer Architecture", courseId: 2, groups: [{name: "CA Squad", groupId: 3}]}
+  const members = [
+    { name: "Shirly fang", school: "UCLA", id: 123, discord: "shirly#123", email: "shirly@gmail.com" },
+    { name: "Shirly fang", id: 123, discord: "shirly#123", email: "shirly@gmail.com" }
   ]
-  const [mainPanelState, setMainPanel] = useState();
 
-  const members= [
-    {name: "Shirly fang", school: "UCLA", id:123, discord:"shirly#123", email:"shirly@gmail.com"},
-    {name: "Shirly fang", id:123, discord:"shirly#123", email:"shirly@gmail.com"}
+  const currentGroup2 = { id: 123, name: "DM Squad", courseName: "Calculus", members: members, day: "friday", time: "4:30pm" }
+
+  const classes2 = [
+    { name: "Discrete Mathematics", courseId: 1, groups: [currentGroup2], },
+    { name: "Computer Architecture", courseId: 2, groups: [currentGroup2] }
   ]
+
+  const groupInformation = [
+    { name: "Group name", value: "" },
+  ];
+
+  const reqs = [{ name: "Al Squad", id: 223, types: "invitation" }, { name: "Calc Gang", id: 223, types: "invitation" }]
+
+  const [requests, setRequests] = useState(reqs);
+  const [currentGroup, setCurrentGroup] = useState(classes2[0].groups[0]);
+  const [classes, setClasses] = useState(classes2);
+
   // Pass this to ClassList
-  function groupClicked(title) {
```

Comment:
can get rid of all of this since its unused. 
Can do the following to setup the networking functionality that will be added later.
```
makeRequest() {

}

getGroup() {
setCurrentGroup(currentGroup2)
}

```